db/migrate/20160613150922_add_event_queue.rb

```diff
@@ -0,0 +1,12 @@
+class AddEventQueue < ActiveRecord::Migration
+  def change
+    create_table :katello_events, :force => true do |t|
+      t.integer :object_id, :null => false
+      t.string :event_type, :null => false
+      t.boolean :in_progress, :default => false, :null => false
+      t.timestamps
+    end
+
+    add_index :katello_events, [:object_id, :event_type, :in_progress], :name => :katello_events_oid_et_ip
```

Comment:
if the table gets really big (for example, if the queue consumer was dead for awhile), you may need an index on `created_at` to speed up finding the next event